lib/src/sync-process/sync-message-port.ts

```diff
@@ -23,18 +23,18 @@ enum BufferState {
    * the buffer to this state so that it can use `Atomics.wait()` to be notified
    * when it switches to `MessageSent`.
    */
-  AwaitingMessage,
+  AwaitingMessage = 0,
   /**
    * The state indicating that a message has been sent. Whenever an endpoint
    * sends a message, it'll set the buffer to this state so that the other
    * endpoint's `Atomics.wait()` call terminates.
    */
-  MessageSent,
+  MessageSent = 1,
   /**
    * The state indicating that the channel has been closed. This never
    * transitions to any other states.
```

Comment:
Document that this is used as a bit mask along with the `AwaitingMessage` and `MessageSent` states.